text/0096-metric-instrument-terminology.md

```diff
@@ -0,0 +1,153 @@
+adjectives, commonly abbreviated "Sync" and "Async", when describing
+instruments.
+2. Prior to OTEP 88, but even with OTEPs 88 and 93 included, there is
+inconsistency in the naming of instruments.  Note that "Counter" and
+"Observer" end in "-er", a noun suffix used in the sense of "[person
+occupationally connected
+with](https://www.merriam-webster.com/dictionary/-er)", while the term
+"Measure" does not fit this pattern.  This proposal proposes to
+replace the abstract term "Measure" by "Recorder", since the
+associated method name (verb) is specified as `Record()`.
+3. The OTEP 93 asynchronous instruments ("LastValueObserver",
+"DeltaObserver", and "CumulativeObserver") have the pattern
+"-Observer", while the OTEP 93 synchronous instruments
+("Counter", "UpDownCounter", "Distribution", "Timing") do not.  This
+proposal keeps "Counter" and "UpDownCounter" for Sum-only synchronous instruments, and does the same
+with "Recorder", yielding "Recorder" and "TimingRecorder".
+4. Confusion over the loss of "Gauge" is addressed by replacing
+"LastValueObserver" with "GaugeObserver".
+
+This proposal also repeats the current specification of the default
+Aggregator for each kind of instrument.
+
+## Explanation
+
+The following table summarizes the four synchronous instruments and
+three asynchronous instruments that will be standardized as a result
+of this set of proposals.
+
+| Existing name | OTEP 93 name       | **Final name**         | Sync or Async | Function | Default aggregation | Measurement kind | Rate support |
+| ------------- | ------------------ | ---------------------- | ----------- | ------------- | ---------- | ---- | --- |
+| Counter       | Counter            | **Counter**            | Sync  | Add() | Sum | Delta | Yes | 
+|               | UpDownCounter      | **UpDownCounter**      | Sync  | Add() | Sum | Delta | No |
+| Measure       | Distribution       | **Recorder**           | Sync  | Record() | MinMaxSumCount | Instantaneous | No |
+|               | Timing             | **TimingRecorder**     | Sync  | Record() | MinMaxSumCount  | Instantaneous | No |
+| Observer      | LastValueObserver  | **GaugeObserver**      | Async | Observe() | MinMaxSumCount  | Instantaneous | No |
+|               | DeltaObserver      | **DeltaObserver**      | Async | Observe() | Sum | Delta | Yes |
+|               | CumulativeObserver | **CumulativeObserver** | Async | Observe() | Sum | Cumulative | Yes |
+
+The argument for "Recorder" instead of "Distribution" is that we
+should prefer instrument descriptives associated with the action being
+performed ("occupationally connected"), not the value being computed,
+as the latter is dependent on SDK configuration.  A "Recorder" records
+a value that is part of a distribution.  A "Counter" counts a value
+that is part of a sum.  An "GaugeObserver" observes an instantaneous
+value ("reads a gauge").  A "Recorder" records an arbitrary value.  A
```

Comment:
It is confusing that "Recorder" is included twice in this list. Once saying it records a distribution and the other saying it records an arbitrary value.  I think the latter is more accurate. If a view of a `Recorder` were to 'aggregate' with an array (i.e. no aggregation) saying a "Recorder" records a value that is part of a distribution, while not incorrect, isn't precise.